### PR TITLE
UML-897 Add links to the grey banner at the bottom of all pages 

### PR DESCRIPTION
**Pull request by Raffers**
## Purpose
Adds additional navigation links within the footer of all pages

Fixes UML-897

## Approach

Adjusts HTML to place links at the bottom of the page, includes route clean up to make them more uniform between actor & viewer. 

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Refunds service_

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

**Comment by Raffers** [comment]
I had already cuaght that & had adjusted my code yesterday. Should see the update within the code now.

File: service-front/app/features/context/UI/AccountContext.php [hunk 1]
```diff
@@ -2204,7 +2204,8 @@ public function iCanSeeTheActorPrivacyNotice()
      */
     public function iAmOnTheActorTermsOfUsePage()
     {
-        $this->ui->visit('/terms-of-use');
+        $this->ui->visit('/create-account');
```
Comment (cooperaj) [inline service-front/app/features/context/UI/AccountContext.php:2207]: We should not be navigating to a page from another unless that is a part of the step. I would say, that given the step definition the original code was more appropriate.

File: service-front/app/features/context/UI/ViewerContext.php [hunk 2]
```diff
@@ -431,7 +431,8 @@ public function iCanSeeTheViewerPrivacyNotice()
      */
     public function iAmOnTheViewerPrivacyNoticePage()
     {
-        $this->ui->visit('/privacy-notice');
+        $this->ui->visit('/terms-of-use');
```
Comment (cooperaj) [inline service-front/app/features/context/UI/ViewerContext.php:434]: Same here

File: service-front/app/src/Actor/src/Handler/ActorTermsOfUseHandler.php [hunk 3]
```diff
@@ -5,18 +5,40 @@
...
 
 use Common\Handler\AbstractHandler;
+use Common\Service\Url\UrlValidityCheckService;
+use Mezzio\Helper\UrlHelper;
+use Mezzio\Template\TemplateRendererInterface;
 use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
 use Laminas\Diactoros\Response\HtmlResponse;
 
 class ActorTermsOfUseHandler extends AbstractHandler
```
Comment (cooperaj) [inline service-front/app/src/Actor/src/Handler/ActorTermsOfUseHandler.php:15]: This handler should be marked as `@codecoverageignore`

File: service-front/app/src/Viewer/src/Handler/ViewerTermsOfUseHandler.php [hunk 4]
```diff
@@ -16,12 +19,31 @@
  */
 class ViewerTermsOfUseHandler extends AbstractHandler
```
Comment (cooperaj) [inline service-front/app/src/Viewer/src/Handler/ViewerTermsOfUseHandler.php:20]: This handler should be marked as `@codecoverageignore`

Comment (Raffers) [reply service-front/app/src/Viewer/src/Handler/ViewerTermsOfUseHandler.php:20]: These files have codecoverageignore on, which have been cut off on the above lines.

File: service-front/app/features/context/UI/AccountContext.php (hunk 1 above)
Comment (Raffers) [reply service-front/app/features/context/UI/AccountContext.php:2207]: Then I'll have to change the step definition, as this is now required for the back button functionalitly to work correctly. So refer header is populated. I did try specifying headers but this didn't seem to work.

File: service-front/app/src/Viewer/src/Handler/ViewerTermsOfUseHandler.php (hunk 4 above)
Comment (cooperaj) [reply service-front/app/src/Viewer/src/Handler/ViewerTermsOfUseHandler.php:20]: Ha. My bad. 👍

File: service-front/app/src/Actor/src/Handler/ActorTermsOfUseHandler.php [hunk 5]
```diff
@@ -5,18 +5,45 @@
...
+    }
+
     /**
      * @param ServerRequestInterface $request
      * @return ResponseInterface
      */
     public function handle(ServerRequestInterface $request): ResponseInterface
     {
-        return new HtmlResponse($this->renderer->render('actor::actor-terms-of-use'));
+        $referer = $this->urlValidityCheckService->setValidReferer($request->getHeaders()['referer'][0]);
```
Comment (johnnolan) [inline service-front/app/src/Actor/src/Handler/ActorTermsOfUseHandler.php:43]: @GemTay what happens if this is null, does it fall back to `/`?

Comment (johnnolan) [reply service-front/app/src/Actor/src/Handler/ActorTermsOfUseHandler.php:43]: Ignore, just checked the code, is does :)

**Pull request merged**